modules/openy_features/openy_block/modules/openy_block_branch_amenities/config/install/paragraphs.paragraphs_type.branch_amenities_with_icons.yml

```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: branch_amenities_with_icons
+label: 'Branch Amenities with Icons'
+icon_uuid: null
+description: ''
```

Comment:
could you add a description for content managers, please

Reply:
Done